samples/Azure.Management.Storage/readme.md

```diff
@@ -17,9 +17,19 @@ modelerfour:
   lenient-model-deduplication: true
   seal-single-value-enum-by-default: true
 
-list-exception:
-- /subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/deletedAccounts/{deletedAccountName}
-- /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/managementPolicies/{managementPolicyName}
+request-path-to-singleton-resource:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/managementPolicies/{managementPolicyName}: managementPolicies/default
+
+request-path-to-parent:
+  /subscriptions/{subscriptionId}/providers/Microsoft.Storage/deletedAccounts: /subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/deletedAccounts/{deletedAccountName}
+  /subscriptions/{subscriptionId}/providers/Microsoft.Storage/checkNameAvailability: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}
+
+operation-positions:
+  /subscriptions/{subscriptionId}/providers/Microsoft.Storage/deletedAccounts: collection
```

Comment:
What would happen if the operation on this path wasn't found?

Reply:
Unfortunately we currently do not have a validation of the paths in the configurations, therefore if this path does not exist, nothing will happen as if this configuration is not here.